frame/nft-fractionalization/README.md

```diff
@@ -0,0 +1,3 @@
+### Lock NFT
+
+Lock an NFT from `pallet-nfts` and mint fungible assets from `pallet-assets`.
```

Comment:
a line or two to explain how it can get unlocked and get access to the unfractionalised nft would be good here (or why that's not possible).